snapshot/git/gitdb/db_test.go

```diff
@@ -54,6 +53,58 @@ func TestIngestDir(t *testing.T) {
 	}
 }
 
+func TestIngestCommit(t *testing.T) {
+	id1, err := fixture.db.IngestGitCommit(fixture.external, fixture.commit1ID)
```

Comment:
Doesn't ingesting commit2 take care of this? Depending on what you want to test this detail could be fine.